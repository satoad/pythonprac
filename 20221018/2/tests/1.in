:sin x sin(x)
sin 1
:decorate s "--<<{}>>--".format(s)
decorate "ЖЖЖ"
sin 2
sin 3
sin 4
:cos x cos(x)
cos 1
quit "{}, {}"