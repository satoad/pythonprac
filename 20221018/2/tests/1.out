0.8414709848078965
--<<ЖЖЖ>>--
0.9092974268256817
0.1411200080598672
-0.7568024953079282
0.5403023058681398
4, 10
